moveit_core/trajectory_processing/src/ruckig_traj_smoothing.cpp

```diff
@@ -441,4 +439,31 @@ void RuckigSmoothing::getNextRuckigInput(const moveit::core::RobotStateConstPtr&
                    ruckig_input.max_acceleration.at(joint));
   }
 }
+
+bool RuckigSmoothing::checkOvershoot(ruckig::Trajectory<ruckig::DynamicDOFs, ruckig::StandardVector>& ruckig_trajectory,
+                                     const size_t num_dof, ruckig::InputParameter<ruckig::DynamicDOFs>& ruckig_input,
+                                     const double overshoot_threshold)
+{
+  // For every timestep
+  for (double time_from_start = 0; time_from_start < ruckig_trajectory.get_duration();
```

Comment:
Is there a feasible way to not start from 0 every time?

Reply:
MoveIt2 has that feature already  :)   I guess I can backport it to MoveIt1

https://github.com/ros-planning/moveit2/pull/1990